bundles/org.openhab.voice.mimictts/src/main/resources/OH-INF/i18n/mimictts.properties

```diff
@@ -0,0 +1,10 @@
+voice.config.mimictts.audioVolatility.label = Audio Volatility
+voice.config.mimictts.audioVolatility.description = The amount of noise added to the generated audio (0-1). Can help mask audio artifacts from the voice model. Multi-speaker models tend to sound better with a lower amount of noise than single speaker models.
+voice.config.mimictts.group.tts.label = TTS Configuration
+voice.config.mimictts.group.tts.description = Parameters for Mimic 3 TTS API.
+voice.config.mimictts.phonemeVolatility.label = Phoneme Volatility
+voice.config.mimictts.phonemeVolatility.description = The amount of noise used to generate phoneme durations (0-1). Allows for variable speaking cadance, with a value closer to 1 being more variable. Multi-speaker models tend to sound better with a lower amount of phoneme variability than single speaker models.
+voice.config.mimictts.speakingRate.label = Speaking Rate
+voice.config.mimictts.speakingRate.description = Controls how fast the voice speaks the text. A value of 1 is the speed of the training dataset. Less than 1 is faster, and more than 1 is slower.
+voice.config.mimictts.url.label = Url
+voice.config.mimictts.url.description = Mimic 3 URL.
```

Comment:
Please add this at the end of this file:
```

# service

service.voice.mimictts.label = Mimic Text-to-Speech
```

Reply:
Done